content/docs/components/multi-tenancy/getting-started.md

```diff
@@ -1,79 +1,28 @@
 +++
-title = "Multi-user Isolation"
-description = "Isolation of user-created resources for convenience and organization"
-weight = 50
+title = "Getting Started with Multi-user Isolation"
```

Comment:
A comment about the last section on this page, **Current Integration and Limitations** - should we move that section to the [Overview page](https://deploy-preview-1682--competent-brattain-de2d6d.netlify.com/docs/components/multi-tenancy/overview/)? (Plus a reminder to change the heading to sentence case as mentioned in a comment below.)